### PR TITLE
Memory game

### PR DESCRIPTION
**Pull request by tavor118sn**
### Memory game

[Demo](https://tavor118sn.github.io/kottans-memory-pair-game/)
[Code base](https://github.com/tavor118sn/kottans-memory-pair-game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/tavor118sn/task_memory_game/script.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+
+const initGame = () => {
+
+  GAME_BOARD.innerHTML = '';
+  pairsFound = 0;
+
+  let allCards = shuffleArray([...CARDS, ...CARDS]);
+
+  allCards.forEach((elem, index) => {
```
Comment (AMashoshyna) [inline submissions/tavor118sn/task_memory_game/script.js:54]: You are manipulating DOM in a loop. DOM access operations are expensive. Let's try to do all changes in one operation. No need to append each card one by one

File: submissions/tavor118sn/task_memory_game/script.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  flipCard.dataset.cardValue = elem;
+
+  const flipInner = document.createElement('div');
+  flipInner.classList.add('flip-card-inner');
+
+  flipInner.innerHTML = `
+    <div class="front">
+      <img class="front-img" src="./img/brawl-stars-main-icon.png" alt="Brawl Icon">
+    </div>
+      <div class="back">
```
Comment (AMashoshyna) [inline submissions/tavor118sn/task_memory_game/script.js:37]: Maybe you can use pseud element here instead?

**Review by AMashoshyna — COMMENTED** [review]
Well done! Just two comments to address.

File: submissions/tavor118sn/task_memory_game/script.js (hunk 1 above)
Comment (tavor118sn) [reply submissions/tavor118sn/task_memory_game/script.js:53]: Fixed

File: submissions/tavor118sn/task_memory_game/script.js (hunk 2 above)
Comment (tavor118sn) [reply submissions/tavor118sn/task_memory_game/script.js:37]: @AMashoshyna Could you give an example? I'm not familiar with CSS and HTML a lot. Only basis.

Comment (tavor118sn) [reply submissions/tavor118sn/task_memory_game/script.js:37]: Just can't imagine how `::after` can be used here

Comment (tavor118sn) [reply submissions/tavor118sn/task_memory_game/script.js]: @AMashoshyna Done

**Comment by AMashoshyna** [comment]
Awesome!

**Pull request merged**